### PR TITLE
Isolate storage library

### PR DESCRIPTION
**Pull request by Awambeng**
Refactor storage logic into a standalone library in a new repository for reusability

### CODE REVIEW

**Comment by nitch2019** [comment]
@hugoib pls check the protection on this repo

**Comment by hugoib** [comment]
Added branch protection, after we set the structure of how many libraries will live in this repo, we will implement proper CI checks:

https://github.com/ADORSYS-GIS/wallet-vc-libs/issues/5https://github.com/ADORSYS-GIS/wallet-vc-libs/issues/5

File: libs/storages/package.json
Comment (hugoib) [inline libs/storages/package.json:17]: I think it is missing typescript, maybe it needs to run:

npm install typescript --save-dev

File: libs/storages/.eslintrc.json
Comment (hugoib) [inline libs/storages/.eslintrc.json:1]: Can we change the name of the folder from storages plural to singular? I think it makes more sense... since it is just one component

**Comment by hugoib** [comment]
I reviewed, it looks good, tests are passing after adding typescript to the package.json

After we approve this:

https://github.com/ADORSYS-GIS/wallet-vc-libs/pull/6

Please adapt to that structure and I will review again before merging

File: libs/storages/package.json
Comment (IngridPuppet) [inline libs/storages/package.json:13]: Shouldn't `@swc/helpers` be a dev dependency?

**Comment by hugoib** [comment]
I reviewed again, looks good! Waiting for the pipeline to pass and then I will approve it.

**Comment by hugoib** [comment]
First merge this: https://github.com/ADORSYS-GIS/wallet-vc-libs/pull/7

**Review by hugoib — APPROVED** [review]

**Pull request merged**
